chronos/transports/stream.nim

```diff
@@ -1595,6 +1643,26 @@ else:
           break
     return retFuture
 
+  proc connect*(address: TransportAddress,
+                bufferSize = DefaultStreamBufferSize,
+                child: StreamTransport = nil): Future[StreamTransport] =
```

Comment:
Could you also add the flags here & pass it to the other connect?

Reply:
Good catch, done.